sockeye/utils.py

```diff
@@ -251,6 +251,28 @@ def variance(self) -> float:
             return self._M2 / self._count
 
 
+def top1(scores: mx.nd.NDArray,
+         offset: mx.nd.NDArray) -> Tuple[mx.nd.NDArray, mx.nd.NDArray, mx.nd.NDArray]:
+    """
+    Get the single lowest element per sentence from a `scores` matrix. Expects that
+    beam size is 1, for greedy decoding.
+
+    NOTE(mathmu): The current implementation of argmin in MXNet much slower than topk with k=1.
```

Comment:
In that case I think we should use topk then and add a TODO to change to argmin once its sped up.

Reply:
The current default behaviour is to use topk for beam size 1, using top1 is a CLI option: `--skip-topk`.

Reply:
but aren't you saying that this is slower than using mx.nd.topk? What's the point of this option if its slower?